public/style/main.css

```diff
@@ -53,7 +86,9 @@ button {
 .fa .pull-right { 
   line-height: inherit;
 }
-
+i.fa {
```

Comment:
this is essentially a global change - let's organize the css better it is hard to read everything is just jumbled